databricks/koalas/frame.py

```diff
@@ -1764,13 +1764,12 @@ def to_latex(
             kdf._to_internal_pandas(), self.to_latex, pd.DataFrame.to_latex, args
         )
 
-    def to_markdown(self, buf=None, mode=None, max_rows=None):
+    def to_markdown(self, buf=None, mode=None):
```

Comment:
Wait .. `to_markdown` is only supported with pandas 1.0.0. @itholic, was `max_rows` supported in pandas 1.0? Seems that parameter was never supported from the first place (https://github.com/pandas-dev/pandas/commit/7ab73a93c94e6557e5d77b2966549955e5c51cbc). Why was this added at https://github.com/databricks/koalas/commit/f44022927f3336ce666f82faa07d7276ed28de05?

Reply:
@HyukjinKwon No, It was just my mistake and I didn't notice that.

Reply:
That's alright. Wanted to double check. Let's merge this as a bug fix.